src/content/getting-started/designers/designers.md

```diff
@@ -25,58 +25,57 @@ title: Designers
 <ul>
     <li><a href="#carbon-design-kit">Carbon Design Kit</a></li>
     <li><a href="#how-to-get-it">How to get it</a></li>
-    <li><a href="#get-the-kit-via-box">Get the kit via Box</a></li>
-    <li><a href="#download-the-kit-from-github">Download the kit from GitHub</a></li>
-    <li><a href="#connect-with-us">Connect with us</a></li>
+    <li><a href="#contribute">Contribute</a></li>
 </ul>
 </anchor-links>
 
 ## Carbon Design Kit
 
-The Carbon Design Kit is a constantly evolving document that contains all of the design system’s visual assets, including components, icons, color palettes, grids, etc. The kit helps you to work faster, and with confidence that your product or experience is aligned with the IBM Design System. It utilizes symbols and text styles, organized via pages in the left side panel of Sketch. This site, along with the Carbon Design Kit, provides designers with everything they need to get up and running with Carbon.
+The Carbon Design Kit is a constantly evolving Sketch document containing all of the design system’s visual assets, including components, icons, color palettes, grids, etc. The kit helps you work faster, with confidence that your product or experience is visually aligned with the IBM Design System. It utilizes symbols and text styles, organized via pages in the left side panel of Sketch. When used in conjunction with this site, the Carbon Design Kit provides designers and developers with everything they need to get up and running with Carbon.
 
-### Release Schedule
+### Release schedule
 
-Every six months the Carbon team releases a major update to the system, including a new design kit, along with new version of the components. Style refinement and bug fixes are released as continuous updates that happen throughout the year. We track all release updates in our [change log](https://github.com/carbon-design-system/carbon-design-kit/releases).
+The Carbon core team releases a major update to the system approximately twice a year, including a revised design kit and new versions of the components when needed. Style refinements and bug fixes are released as continuous updates throughout the year. All design kit release updates are tracked in the [change log](https://github.com/carbon-design-system/carbon-design-kit/releases).
 
 ## How to get it
 
-There are three ways to obtain the kit. For timely updates, install the kit as Sketch Shared Library. Please note this requires Sketch version 53 or above.
+There are three ways to obtain the kit. For timely updates, install the kit as Sketch Shared Library. _Please note: this method requires Sketch version 53 or above._
```

Comment:
This version number will get out of date quickly, I think just saying "requires the latest version of Sketch" will suffice.

Reply:
I see where you're coming from here, but the requirement is technically version 53, not the newest version of Sketch. E.g., if the latest version is v54, v53 will still suffice for this method, right? If someone is stuck with v53, they can still use this method.

Reply:
Its on Sketch 53 until Sketch 54 comes out then the requirement will be Sketch 54. They do updates on a pretty regular basis.

Reply:
Actually, on second thought this note applies at all of the methods not just the Shared Library.

Reply:
OK, I must be confused about how this requirement works. . . Is it that the user's version of Sketch must match the version owned by the creator of the master file? Or is 53 the version where this capability was introduced? Thanks :)

Reply:
The user's version must match the version the master kit was made on. I'm not sure if its just major version or if it has to match minor versions as well to be compatible.

Reply:
I get these errors pretty often (not just with 53), so I think just sticking to `latest` makes the most sense, as updating my sketch to match the kits version usually resolves it

Reply:
Yep, we decided to make that change. Just updated.